nodes/node.go

```diff
@@ -23,35 +24,25 @@ const (
 var NodeKind string
 
 const (
-	NodeKindBridge           = "bridge"
-	NodeKindCEOS             = "ceos"
-	NodeKindCVX              = "cvx"
-	NodeKindCRPD             = "crpd"
-	NodeKindHOST             = "host"
-	NodeKindLinux            = "linux"
-	NodeKindMySocketIO       = "mysocketio"
-	NodeKindOVS              = "ovs-bridge"
-	NodeKindSonic            = "sonic-vs"
-	NodeKindSRL              = "srl"
-	NodeKindVrCSR            = "vr-csr"
-	NodeKindVrPAN            = "vr-pan"
-	NodeKindVrN9KV           = "vr-n9kv"
-	NodeKindVrFTOSV          = "vr-ftosv"
-	NodeKindVrROS            = "vr-ros"
-	NodeKindVrSROS           = "vr-sros"
-	NodeKindVrVEOS           = "vr-veos"
-	NodeKindVrVMX            = "vr-vmx"
-	NodeKindVrVQFX           = "vr-vqfx"
-	NodeKindVrXRV            = "vr-xrv"
-	NodeKindVrXRV9K          = "vr-xrv9k"
-	NodeKindVrNXOS           = "vr-nxos"
-	NodeKindIPInfusionOCNOS  = "ipinfusion-ocnos"
-	NodeKindKeysightIxiaCOne = "keysight_ixia-c-one"
+	NodeKindBridge = "bridge"
+
+	NodeKindHOST = "host"
+	NodeKindOVS  = "ovs-bridge"
+	NodeKindSRL  = "srl"
```

Comment:
thanks @steiler this change LGTM!
I think we can chase those dependencies in config.go in a separate PR